pkg/plugin/stream.go

```diff
@@ -54,8 +54,8 @@ func (a *App) runOpenAIChatCompletionsStream(ctx context.Context, req *backend.R
 	if err != nil {
 		return err
 	}
-	path := strings.TrimPrefix(req.Path, "openai")
-	httpReq, err := http.NewRequestWithContext(ctx, http.MethodPost, settings.OpenAI.URL+path, bytes.NewReader(outgoingBody))
+	httpPath := strings.TrimPrefix(req.Path, "openai")
```

Comment:
This will have the UUID generated by the frontend on it now, which we don't want to include in the path we use on the request to OpenAI. I think this would actually suffice:

```suggestion
	httpPath := strings.TrimPrefix(openAIChatCompletionsPath, "openai")
```

(or some static version so we don't need to do it every request, but either works)